logging.md

```diff
@@ -0,0 +1,34 @@
+# Conventions for LOGGING
+[See issues](https://github.com/OCR-D/spec/issues/11)
+All logging may be stored in files. 
+If so, they must be given in Provenance as a result. 
```

Comment:
Provenance capitalized sounds kind-of biblical :) Seriously though, I'm missing context here, I don't understand what `if` is related to, what `being given in provenance` means and what a `result` is here.